en/docs/reference/pricing.md

```diff
@@ -2,229 +2,142 @@
-
-#### Pay As You Go (PAYG) 
-Pay As You Go is a flexible usage-based payment plan. While the first 5,000 steps are free per month, Choreo charges you $0.005 for each additional step based on your usage per month. For example, for 1,000 steps Choreo charges you $5. 
-
-Let’s assume you have used a total of 12,500 [steps](#what-is-a-step) for a particular month, your monthly usage for that month under the PAYG plan will be:
-
-    Chargeable steps usage: 12,500 - 5,000 = 7,500
-    The total charge for the usage: 7,500 * $0.005 ($5 for every 1,000 steps) = $37.50
-
-#### Enterprise 
-The Enterprise plan is suitable for large-scale usage of Choreo. You can access volume discounts and committed-use discounts with this plan. [Contact sales](https://wso2.com/contact/?ref=choreo) to get a discounted rate.
-
-### What is a step?
-A step (1 step) is a maximum of 500ms compute time on a container having 350MB of memory and 0.5 vCPU. Any additional compute time is counted as an additional step.  
-An incoming event or message to Choreo or an outgoing API call from Choreo is a minimum of 1 step.
+The pricing for your Choreo deployment is mainly determined based on the following:
 
+- **The number of components**
 
-## Pricing examples
+    You can have up to five free components ((i.e., any combination of component types), and pay for each additional component.
 
-### Example 1: REST API 
+- **Infrastructure cost**
 
-An organization has a flight booking system on Choreo. This system involves several APIs and databases. The scenario and the APIs being used in the system are described below.
+    The infrastructure cost consists of the charges to Choreo by the cloud platform for vCPU, RAM, storage, ingress, egress, and network. Choreo provides $100 worth of infrastructure free of charge to each organization.
 
-#### Scenario
+    If the infrastructure cost of an organization on the Free plan exceeds $100, Choreo will limit some of the services it provides to it (e.g., some components might be deactivated).
 
-To make a booking through the web application, a client first searches for available flights and then proceeds to make the booking. The client application (web application) upon receiving the request makes a call to the [Search API](#searchAPI) to search for available flights. When the client receives the search results the client proceeds to make a reservation using the [Reservations API](#reservationsAPI).
+    If the infrastructure cost of an organization on the Pay As You Go plan exceeds $100, it can pay the additional infrastructure charges to the Choreo Sales Team.
 
-The following two diagrams illustrate the interactions in the system while searching and reserving a flight.
-<br/>
+    If the infrastructure cost of an organization on the Enterprise plan exceeds $100, it can pay the additional infrastructure charges to the Choreo Sales Team or directly to the cloud platform vendor.
```

Comment:
For the enterprise plans, payments are handled by the sales team, not the cloud vendor (i.e. Stripe, Azure). @isuruw please verify.

Reply:
Changed this.

Reply:
+1